vllm/entrypoints/openai/api_server.py

```diff
@@ -62,12 +67,12 @@ async def lifespan(app: fastapi.FastAPI):
     async def _force_log():
         while True:
             await asyncio.sleep(10)
-            await engine.do_log_stats()
+            await backend.do_log_stats()
```

Comment:
I think this should just move into AsyncLLMEngine right? (perhaps whether it's started controlled by some arg)

Reply:
Fine with me if it works